### DISCUSSION TITLE
[Scam Alert] Iran Coinmine ایران کوین ماین

### DISCUSSION THREAD

**Discussion by saeedkousha** [Webamoozcom/warning-list#87] [Announcements]
<div dir="rtl">

**Project Name:** 
Iran Coinmine

**نام پروژه:** 
ایران کوین ماین

**نماد پروژه:**
IC

**اتفاقی که افتاده است:**

![](https://cdn.arzdigital.com/uploads/2019/04/irancoinmine.jpg)

این روزها کلاهبرداری‌های مربوط به ماینینگ یا استخراج ابری با سودهای تضمینی و نجومی، به شدت در حال افزایش هستند. پس از راه‌اندازی شدن سایت کلاهبرداری مستر دیجی کوین، حالا وب‌سایت دیگری به نام «ایران‌کوین‌ماین» به آدرس https://irancoinmine.com ظهور کرده است. در این مقاله به بررسی این پروژه کلاهبرداری می‌پردازیم.

شاید شما هم با دیدن سودهای نجومی استخراج دوج‌ کوین در سایت‌های مختلف، بدون اینکه لازم باشد خودتان تجهیزاتی بخرید و نگران هزینه برق یا خرابی‌ها و سر و صدا و مشکلات دستگاه استخراج خود باشید، دوست داشته باشید سرمایه خود را در اختیار یکی از این سایت‌هایی که ادعا می‌کنند با استخراج دوج‌ کوین می‌توانند سودهای بالایی به شما بدهند، قرار دهید.

![](https://irancoinmine.com/website/images/dashboard.png)

هر جا سود تضمینی با نرخ‌های نجومی دیدید به کلاهبرداری بودن آن شک کنید، مخصوصا اگر در فضای غیرقانون‌گذاری شده ارزهای دیجیتال و ماینینگ ابری باشد.

آیا منطقی ست سود این سایت از سودی که تعهد کرده به سرمایه‌گذاران می‌دهد کمتر باشد؟ با یک حساب سر انگشتی و با در نظر گرفتن سودی که سایت برای خود در نظر گرفته و همچنین هزینه برق و نگهداری و تعمیرات دستگاه‌ها ، استخراج دوج‌کوین باید حداقل ۲ برابر ارقام ارائه شده برای خودشان سود داشته باشد تا بخواهند این سودها را به سرمایه‌گذاران بدهند، یعنی حداقل ۲۰۰% سود! یادتان باشد همیشه برنده نهایی این سایت‌ها هستند.

در بخش دیگری از سایت قید شده که همه سرمایه‌گذاری آنها در بخش استخراج و آن هم استخراج دوج‌کوین نیست، بر اساس ادعای این سایت، توزیع سرمایه‌گذاری آنها در بخش های مختلف به صورت زیر است:

%۴۰ سرمایه گذاری در بازار بورس بین المللی نیویورک(استاک مارکت)
%۴۰ از طریق ماینینگ ارزهای پر سود
%۲۰ ترید کردن ارزهای دیجیتال
بنا بر ادعای این سایت تنها %۴۰ از سرمایه جذب شده توسط آنها به بخش استخراج اختصاص می یابد که همان هم بین استخراج ارزهای مختلف تقسیم می‌شود. سوال اینجاست که دلیل ارائه ندادن اطلاعات شفاف توسط این سایت چیست؟ چرا بطور واضح بیان نشده که استخراج چه ارزهایی صورت می‌گیرد؟

با وجود اینکه همه ارزها با تجهیزات یکسانی استخراج نمی شوند و همچنین نرم‌افزار لازم برای استخراج ارزهای مختلف نیز متفاوت است، تغییر دادن ارز دیجیتالی که استخراج می‌شود نیز ساده و بدون هزینه نخواهد بود و در بسیاری از مواقع توجیح اقتصادی ندارد، بنابراین ارائه ندادن لیستی برای ارزهایی که استخراج می شوند، می‌تواند نشانه‌ای باشد از این امر که اصلا استخراج و تجهیزات ماینینگی وجود ندارد.

همچنین نام بردن از دو عنوان کلی سرمایه‌گذاری در بازار بورس بین المللی نیویورک و ترید کردن ارزهای دیجیتال که ۶۰% کل سرمایه‌گذاری‌های آنها را در برمیگیرد نیز بسیار بحث برانگیز است.

سود بردن در بازار بورس بین المللی نیویورک به دانش و تجربه بسیار بالایی نیاز دارد، و ترید کردن ارزهای دیجیتال نیز علاوه بر نیاز به تسلط کامل بر تحلیل تکنیکال و فاندامنتال ارزهای مختلف بازار بسیار پر ریسکی می‌باشد. بنابراین قطعا نباید سود تضمینی و با چنین نرخی ارائه شود.

حق تام الاختیار ایران‌کوین‌ماین در برابر سرمایه افراد
دیگر موضوع بحث‌ برانگیزی که در بخش شرایط و ضوابط سایت آمده، داشتن اختیار کامل ایران‌ کوین‌ ماین برای تغییر شرایط و قوانین است. بر اساس این ماده مشتریان این حق را به ایران‌کوین‌ماین می‌دهند که هر زمان که خواست هر یک از قوانین سایت را که تمایل داشت تغییر دهد. این عملا به معنای دادن وکالت تام‌الاختیار سرمایه خود به آنهاست زیرا بر اساس این شرط، ایران‌کوین‌ماین حق دارد هر زمان که بخواهد قوانین را به نفع خود تغییر دهد و مشتریان نیز حق هیچ گونه اعتراض و شکایتی ندارند، زیرا این آیین‌نامه را در ابتدا پذیرفته‌اند.

متن این شرط از آیین‌نامه ایران‌کوین‌ماین عینا در زیر آمده است:

«ایران کوین ماین حق دارد در هر زمان شرایط و قوانین را تغییر دهد، همچنین ایران کوین ماین مختار به عدم ادامه بازی بوده و میتواند هر زمان که بخواهد به آن خاتمه دهد ، از طرفی ایران کوین ماین موظف است حداقل ۳۰ روز قبل از اتمام بازی آن را اطلاع رسانی نماید تا مشتریان خود تصمیم به ادامه بازی یا عدم ادامه بازی بگیرند. درصورتی که مشتری تمایل به پذیرفتن تغییرات یا اضافه شدن ندارد ، حق دارد تا موافقت نامه را تا تاریخی که صلاح می‌داند اجرا نماید و اختیار خروج از وبسایت را بدون هیچ محدودیتی دارد.»

موضوع دیگری که درباره قانون بالا جلب توجه می‌کند، بکار بردن واژه «بازی» از سوی سایت و اختیار ادامه دادن یا توقف بازی از سوی آن‌ها در هر زمانی که بخواهند است. سرمایه‌گذاری در بورس نیویورک، ترید کردن ارزهای دیجیتال و همچنین استخراج این ارزها هیچ کدام چیزی نیست که بتوان از آن به عنوان بازی یاد کرد. کار کردن در هر کدام از این سه بخش نیاز به علم و تجربه زیادی دارد، و یک کار کاملا تخصصی به حساب می‌آید و به هیچ عنوان نمی‌توان به عنوان به جنبه سرگرمی و بازی نگاه کرد مگر اینکه تعهدی در میان نباشد و حفظ سرمایه‌ای که وارد آن کار شده اهمیت نداشته باشد.

براساس اطلاعات سایت های رتبه بندی بین المللی 96 درصد از بازدیدهای سایت از داخل کشور صورت گرفته است. همچنین دامنه سایت سال گذشته به ثبت رسیده و سرور آن در کشور انگلستان قرار دارد.
بررسی اطلاعات فنی سایت نشان می دهد با آنکه این سایت به عنوان سپرده پذیر با بازدهی بالا فعالیتش را آغاز کرده اما در عمل فعالیت مولدی ندارد. مدیران و عوامل سایت از تجارت هایی چون بورس بین المللی نیویورک نام می برند در حالیکه به دلیل تحریم های اعمال شده بر کشورمان چنین ادعایی امکان پذیر نیست. 
پرداخت سود کلان  ناشی از سرمایه گذاری در زمینه ارزهای دیجیتال نیز به نوعی خلاف واقع و غیرمنطقی است زیرا سختی های استخراج رمز ارز و هزینه بالای نگهداری و برق مصرفی آن عملاً سودآوری بالا را  ممکن نخواهد کرد.
تأسیس سایت سرمایه گذاری بدون ارائه طرح منطقی این نتیجه را متبادر می سازد که سودهای پرداختی از اصل سرمایه سپرده گذاران بوده و در عمل پانزی بودن طرح شرکت مذکور را آشکار می سازد. 

**قوانینی که نقض شده است:**

**نوع هشدار:**

کلاهبرداری

**نوع کلاهبرداری:** 

کلود ماینینگ
پانزی

**وضعیت کلاهبرداری:**
در جریان

**نام کلاهبرداران:**

**نام شرکت(ها):**

**آدرس وب سایت(ها):**

https://irancoinmine.com/

**برآورد مبلغ کلاهبرداری:**

**منابع:**

[ایران هشدار](https://www.iranhoshdar.ir/Default.aspx?PageName=Newsb&ID=40876&Language=1&tite=%DA%A9%D9%84%D8%A7%D9%87%D8%A8%D8%B1%D8%AF%D8%A7%D8%B1%DB%8C-%D9%BE%D8%A7%D9%86%D8%B2%DB%8C-%D8%A7%DB%8C%D9%86-%D8%A8%D8%A7%D8%B1-%D8%A8%D8%A7-%D9%86%D8%A7%D9%85-%D8%A7%DB%8C%D8%B1%D8%A7%D9%86-%DA%A9%D9%88%DB%8C%D9%86-%D9%85%D8%A7%DB%8C%D9%86!!)

[ارز دیجیتال](https://arzdigital.com/irancoinmine-scam-review/)

**جمع بندی:**

**Comment by Shahriar57** [comment]
این سایتی که اعلام کردید در اینستا گرام پیج فعال داره، 
https://www.instagram.com/irancoinmineofficial/
در سایت ساماندهی هم اطلاعات در حال بررسی هست 
صاحب امتیاز: علی اکبر توسل
پست الکترونیکی: alimahbublove@gmail.com
